org.openhab.binding.zigbee/README.md

```diff
@@ -46,6 +46,8 @@ The firmware can be flashed with `./cc-tool -e -w CC2531ZNP-Pro-Secure_Standard.
 
 The Ember EZSP NCP (Network Co-Processor) supports the Silabs EM358 or MightyGecko dongles with the standard NCP firmware. The thing type is ```coordinator_ember```.
 
+Note that there are generally two versions of the Ember NCP firmware in use. One operates at a baud rate of 115k2 with RTS/CTS flow control (i.e. hardware flow control), the other operates at a baud rate of 57k6, and XON/XOFF flow control (i.e. software flow control). If you are programming your own stick (eg the CEL stick) then it should be advisable to use the hardware flow control version - many commercial sticks seem to use the lower speed and software flow control (e.g. Bitron and Nortek HUSBZB-1).
```

Comment:
Just a minor comment

once you wrote eg and once you wrote e.g. - not sure if this is a  typo or a language thing :D